vng_api_common/constants.py

```diff
@@ -8,6 +8,9 @@
 BSN_LENGTH = 9
 RSIN_LENGTH = 9
 
+RSIN_LIST_SIZE = [8, 9]
+BSN_LIST_SIZE = [8, 9]
```

Comment:
@sergei-maertens do you know if 8 numbers is also allowed for these? I seem to find conflicting information when looking this up
- https://www.rijksoverheid.nl/onderwerpen/privacy-en-persoonsgegevens/vraag-en-antwoord/wat-is-het-burgerservicenummer-bsn mentions 9 for BSN
- https://www.nederlandwereldwijd.nl/bsn/wat-is-bsn mentions 8 or 9 for BSN

For RSIN I only see 9 mentioned

Reply:
I think historically SOFI number could be 8 digits, but as far as I understand this is phased out entirely and only 9 digit numbers are allowed. I haven't gotten any error reports yet from places that don't support 8 chars, so I'd just try sticking to 9. But, maybe @alextreme has more insights with his experience from BD?

Reply:
Naturally: in the same category as EBDIC and CSV files, voorloopnullen for 8-digit BSN's are the bane of BD. e.g. 011122233 != 11122233

I'd want to avoid 8-digit BSNs as much as possible, but if this is what I think it is then it may be too late...